### ISSUE TITLE
Add Functionality for Opening CCS Session

### ISSUE THREAD

**Issue by webbcam** [webbcam/tiflash#17]
Add functionality for opening/attach CCS session.

## Context

The Debug Server provides the ability to launch a CCS Session. This can be done after performing any actions (ie flash, erase, etc) and then CCS will launch the Debug Session (connected to the device).

## Process

Opening a CCS session is done by calling:
getServer("CCSServer.1");

## Possible Fix

To do this, add an "attach" option to the session args. If this attach option is provided, tiflash will perform the specified actions as normal, but instead of ending it will open and attach a CCS session.
TIFlash will return once the CCS Session is closed.

1. Add an "attach" argument to session arguments (core.py args.py
2. Check for an attach argument in js/main.js
3. Launch getServer("CCSServer.1") if "attach" argument present

----

**Comment by webbcam** [comment]
Closing for now; will open new issue for next implementation

**Comment by webbcam** [comment]
First implementation:

    Allows user to attach a CCS session to device either after running a command or as a command itself
    To end the CCS session, the user must press enter in the python prompt/cli (see below)

The current way to end the CCS session (by pressing enter on python prompt/cli) is a bit hacky and therefore not ideal. In many instances the user could easily press enter in the cli on accident in the middle of using the CCS session and end the session altogether.

The ideal solution would be for the script to finish (and CCS session to end) when the user closes out of the CCS session by directly closing the application/GUI.

The current implementation does not do this because:

    in order to keep the CCS session open, the javascript (and thus python script) cannot end. Therefore to keep the javascript running, it needs to wait on something to finish.
    originally the thought was to have the javascript wait on the CCS session thread to complete and then close once it finishes. However it turns out when the CCS GUI is closed, the CCS thread does not end. Therefore the javascript would continue waiting forever.

I want to come back to this in the future to try to come up with a better solution
